gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/cloud/storage/contrib/nio/it/ITGcsNio.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I checked and there are 248 files that say "Copyright 2015 Google". Perhaps we may want to file an issue to remember to update those periodically. Or that'll just be taken care of by the automated header check.